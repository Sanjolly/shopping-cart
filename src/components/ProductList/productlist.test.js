import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ProductList from './ProductList';

configure({ adapter: new Adapter() });

describe('ProductList', () => {

    const productlist = shallow(<ProductList />);

    it('renders correctly', () => {
        expect(productlist).toMatchSnapshot();
    });

});