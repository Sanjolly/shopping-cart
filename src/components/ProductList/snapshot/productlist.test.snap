// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`productlist renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Text />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "bsStyle": "default",
      "children": <li
        class="custom"
>
        productlist
</h1>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "productlist",
        "class": "custom",
      },
      "ref": null,
      "rendered": "No productlist",
      "type": "li",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "bsStyle": "default",
        "children": <li
          class="custom"
>
          productlist
</h1>,
     },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "productlist",
          "class": "custom",
        },
        "ref": null,
        "rendered": "No productlist",
        "type": "li",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
